index.js

```diff
@@ -1,7 +1,15 @@
 /* 
-    33 JavaScript Concepts is a project created to help JavaScript developers master their skills. It is a compilation of fundamental JavaScript concepts that are important and fundamental. 
+    33 JavaScript Concepts is a project created to help JavaScript developers master their skills. 
+    
+    It is a compilation of fundamental JavaScript concepts that are important and fundamental. 
 
-    This project was inspired by an article written by Stephen Curtis. 
+    JavaScript is a powerful and a very simple language. This language is used to make tasks automation, Web Pages, Servers, Desktop Apps, Databases, etc.
+    
+    Learning the fundamental JS concepts is very important to know how to use this so strong language.
```

Comment:
It's crucial to understand the foundational JS ideas in order to use this powerful language.